src/components/ArticleCard.jsx

```diff
@@ -1,10 +1,17 @@
-const ArticleCard = ({title, article_img_url, votes}) => {
-    
-    return <section className="article-thumbnails">
-        <strong>{title}</strong><br /><br />
+import { Link } from "react-router-dom";
+
+const ArticleCard = ({ title, article_img_url, votes, article_id }) => {
+
+  return (<Link to={`articles/${article_id}`}>
+      <div className="article-thumbnails">
+        <strong>{title}</strong>
```

Comment:
`<strong>` shouldn't just be used in place of bold, it's more for people using screen readers for things of importance like errors or warnings, `<b>` works just fine here